src/debugger/sourceMap.ts

```diff
@@ -15,7 +15,7 @@ interface ISourceMap {
 }
 
 export class SourceMapUtil {
-    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+?)\s*$/m;
+    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+index\.ios\.map.*?)\s*$/m;
```

Comment:
If the script body returns multiple ```sourceMappingUrl=``` then this error happens:
```
[6:39:12 PM] <END>   request:/**/index.ios.bundle?platform=ios (67ms)
[6:39:12 PM] <START> request:/exp.js.map
Error: Error validating module options: child "platform" fails because ["platform" must be a string]
    at declareOpts.js:60:13
    at index.js:222:20
    at tryCallOne (/Users/mark/**/relax/node_modules/promise/lib/core.js:37:12)
    at /Users/mark/**/node_modules/promise/lib/core.js:123:15
    at flush (/Users/mark/**/node_modules/asap/raw.js:50:29)
    at _combinedTickCallback (node.js:370:9)
    at process._tickCallback (node.js:401:11)
```

```bash
$ grep "sourceMappingURL=" index.ios.bundle
//# sourceMappingURL=exp.js.map
//# sourceMappingURL=cldr.js.map
//# sourceMappingURL=core.js.map
//# sourceMappingURL=en.js.map
//# sourceMappingURL=main.js.map
//# sourceMappingURL=utils.js.map
//# sourceMappingURL=es5.js.map
//# sourceMappingURL=compiler.js.map
//# sourceMappingURL=en.js.map
//# sourceMappingURL=main.js.map
//# sourceMappingURL=diff.js.map
//# sourceMappingURL=es5.js.map
//# sourceMappingURL=core.js.map
//# sourceMappingURL=es5.js.map
//# sourceMappingURL=memoizer.js.map
//# sourceMappingURL=core.js.map
//# sourceMappingURL=parser.js.map
//# sourceMappingURL=index.ios.map
```

The  only source map of interest is the index.ios/android.map, right? Or should we trust that the correct one is the last match?